### ISSUE TITLE
500 errors occur after a certain short timeout when using Chrome

### ISSUE THREAD

**Issue by allanlei** [swaglive/dockerfiles#1]
```
2018/12/10 04:18:28 [error] 10#10: *15 lua entry thread aborted: runtime error: content_by_lua(default.conf:48):4: Expected value but found invalid token at character 1
stack traceback:
coroutine 0:
	[C]: in function 'decode'
	content_by_lua(default.conf:48):4: in function <content_by_lua(default.conf:48):1> while sending to client, client: unix:, server: , request: "GET /authorize HTTP/1.0", host: "credentials"
2018/12/10 04:18:28 [error] 10#10: *13 auth request unexpected status: 500 while sending to client, client: 10.28.0.115, server: , request: "GET /asia.contents.swag.live/messages/584e2effd51f7c00113d3d45.mp4 HTTP/1.1", host: "storage.google-cloud.svc.cluster.local"
```
but when using `curl`, does not occur.

Also, using `curl` when the error is occurring in Chrome will fix the error in Chrome.

Must be header/caching related

```
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8
Accept-Encoding: gzip, deflate
Accept-Language: en-CA,en;q=0.9,zh-TW;q=0.8,zh;q=0.7
Cache-Control: max-age=0
Connection: keep-alive
Host: storage.google-cloud.svc.cluster.local
Upgrade-Insecure-Requests: 1
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36
```

```
$ curl -v http://storage.google-cloud.svc.cluster.local/asia.contents.swag.live/messages/583d5195f6f58100152a0008.128x128.jpg
*   Trying 10.32.6.234...
* TCP_NODELAY set
* Connected to storage.google-cloud.svc.cluster.local (10.32.6.234) port 80 (#0)
> GET /asia.contents.swag.live/messages/583d5195f6f58100152a0008.128x128.jpg HTTP/1.1
> Host: storage.google-cloud.svc.cluster.local
> User-Agent: curl/7.62.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Server: openresty/1.13.6.2
< Date: Mon, 10 Dec 2018 04:21:38 GMT
< Content-Type: image/png
< Content-Length: 12002
< Connection: keep-alive
< ETag: COPXsq3rtN0CEAE=
< Cache-Control: no-cache, no-store, max-age=0, must-revalidate
< Pragma: no-cache
< Expires: Mon, 01 Jan 1990 00:00:00 GMT
< X-Cache: MISS
```

**Comment by allanlei** [comment]
Seems Metadata server doesn't like 
```
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36
Accept-Encoding: gzip, deflate
```
Dropping either `User-Agent` or `Accept-Encoding` will resolve the 500 error.

Best solution is to drop the request headers
```
proxy_pass_request_headers off;
proxy_pass_request_body off;
```

**Comment by allanlei** [comment]
Fixed in 38ab4b60f9307fe54e54c7e71ea122a9c74925dd
